tests/system/providers/amazon/aws/example_gcs_to_s3.py

```diff
@@ -38,18 +50,40 @@
 ) as dag:
     test_context = sys_test_context_task()
     env_id = test_context["ENV_ID"]
+    gcp_user_project = test_context[GCP_PROJECT_ID]
 
     s3_bucket = f"{env_id}-gcs-to-s3-bucket"
     s3_key = f"{env_id}-gcs-to-s3-key"
 
     create_s3_bucket = S3CreateBucketOperator(task_id="create_s3_bucket", bucket_name=s3_bucket)
 
+    gcs_bucket = f"{env_id}-gcs-to-s3-bucket"
+    gcs_key = f"{env_id}-gcs-to-s3-key"
+
+    create_gcs_bucket = GCSCreateBucketOperator(
+        task_id="create_gcs_bucket",
+        bucket_name=gcs_bucket,
+        resource={"billing": {"requesterPays": True}},
```

Comment:
To test that GCSToS3Operator works with requester pays buckets, you need a bucket with "requester pays" enabled. Luckily, you don't need two GCP accounts to use this feature. You can specify your own GCP project ID as the billable project. If a bucket has "requester pays" enabled and you don't specify a `user_project`, an error will occur.